src/rdkafka_fetcher.c

```diff
@@ -616,6 +742,10 @@ rd_kafka_fetch_reply_handle(rd_kafka_broker_t *rkb,
         const int log_decode_errors = LOG_ERR;
         rd_kafka_topic_t *rkt       = NULL;
         int16_t ErrorCode           = RD_KAFKA_RESP_ERR_NO_ERROR;
+        uint64_t _tagcnt;
+        uint64_t _tag, _taglen;
```

Comment:
these won't be needed when using `rd_kafka_buf_read_tags`
```suggestion
```

Reply:
Removed